addons/mail/static/src/chatter/web/form_compiler.js

```diff
@@ -37,7 +33,7 @@ function compileChatter(node, params) {
     const chatterContainerXml = createElement("t");
     setAttributes(chatterContainerXml, {
         "t-component": "__comp__.mailComponents.Chatter",
-        hasActivities,
+        hasActivities: Boolean(params.hasActivities),
```

Comment:
Does this still have to be done in form compiler?
Probably not as it no longer depends on the template itself.

This should directly read the `has_activities` value on the model instead.

I don't know the exact syntax but something like
`"__comp__.props.model.has_activities"`